packages/common/src/components/types.ts

```diff
@@ -1,14 +1,17 @@
 import { FilterDef } from './Filter/types';
 
+type OpenAPIjsonPath = string | ((resourceData: unknown) => unknown);
+
 export interface SortType {
   isAsc: boolean;
-  id: string;
-  toLabel(t: (key: string) => string): string;
+  resourceFieldID: string;
+  label: string;
 }
 
-export interface Field {
-  id: string;
-  toLabel(t: (key: string) => string): string;
+export interface ResourceField {
```

Comment:
hmm, didn't find documentation about which naming convention is best, `userID` or `userId`
 changing as suggested because I didn't find any hard convention we should use
 @rszwajko  do you have documentation of this naming convetions?
 
 addressed in https://github.com/kubev2v/forklift-console-plugin/pull/281

Reply:
I don't think there is a hard rule in JS world. I recall this linter rule from Java world although old code (pre-Java 8) was not very consistent in this aspect. Example of checkstyle rule: https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s5.3-camel-case